demos/formula_recognition_demo/python/README.md

```diff
@@ -59,7 +59,7 @@ Regardless of what mode is selected (interactive or non-interactive) the process
 Sympy python package is used for rendering. To install it, please, run:
 `pip install -r requirements.txt`
 Sympy package needs LaTeX system installed in the operating system.
-For Windows you can use MiKTeX (just download and install it), for Ubuntu/MacOS you can use TeX Live:
+For Windows you can use [MiKTeX](https://miktex.org/) (just download and install it), for Ubuntu/MacOS you can use TeX Live:
```

Comment:
for Ubuntu/MacOS you can use TeX Live --> for Ubuntu and macOS you can use TeX Live*

Reply:
done